### PR TITLE
fix #78092 hltv.org

### LINKED ISSUE #78092: www.hltv.org
### Issue URL (Ads)
 [https://www.hltv.org/](https://adguardteam.github.io/AnonymousRedirect/redirect.html?url=https%3A%2F%2Fwww.hltv.org%2F)

### Screenshots
<details>
  <summary>Screenshot 1</summary>

  Screenshot 1
</details>

### System configuration

Information | value
--- | ---
AdGuard product: | AdGuard Browser extension v3.5.31
Browser: | Chrome
Stealth mode options: | Hide your search queries, <br>Send Do-Not-Track header, <br>Remove X-Client-Data header from HTTP requests, <br>Strip URLs from tracking parameters, <br>Self-destructing third-party cookies (2880), <br>Block WebRTC, <br>Hide your Referrer from third-parties
Filters: | <b>Ad Blocking:</b><br/> AdGuard Base<br/><br/><b>Privacy:</b><br/> AdGuard Tracking Protection<br/><br/><b>Social Widgets:</b><br/> AdGuard Social Media<br/><br/><b>Annoyances:</b><br/> AdGuard Annoyances<br/><br/><b>Other:</b><br/> Filter unblocking search ads and self-promotion<br/><br/><b>Language-specific:</b><br/> AdGuard Turkish, <br/>AdGuard Russian, <br/>AdGuard Spanish/Portuguese, <br/>Dandelion Sprout's Nordic Filters
Other extensions used: | Нету

### PR DESCRIPTION
**Pull request by wout-codes**
#78092

Links: 
https://www.hltv.org/news/31466/vod-hltv-confirmed-with-guest-styko-talking-fpx-igl-situation-epl-playoffs-end-of-cloud9
https://www.hltv.org/news/31426/ex-north-trio-reunite-in-new-team

### CODE REVIEW

File: EnglishFilter/sections/specific.txt [hunk 1]
```diff
@@ -263,6 +263,8 @@ feetporno.com,familyporn.tv,freehardcore.com,webvoyeur.com,moviesand.com,momvids
 !
 !
 !
+||kittyglitter-*/brand/specbonus-image/bet365/*.png
```
Comment (Alex-302) [inline EnglishFilter/sections/specific.txt:266]: please gise url of the page. Cannot find where these rules can be applied https://github.com/AdguardTeam/AdguardFilters/pull/78967/files#diff-8e737f6429801c4aac12f0a8ddb7b4ab3173f39642bfe56004e22dede37edd6dR266-R267

scriptlet is ok

Comment (wout-codes) [reply EnglishFilter/sections/specific.txt:266]: `@stanislav-atr` added it. I have two more rules which he has missed.

Comment (zloyden) [reply EnglishFilter/sections/specific.txt:266]: @Alex-302 most of changes was merged from https://github.com/AdguardTeam/AdguardFilters/pull/78509.

Comment (wout-codes) [reply EnglishFilter/sections/specific.txt:266]: <details> 

image
</details>

Comment (wout-codes) [reply EnglishFilter/sections/specific.txt:266]: @zloyden @Alex-302 He missed two things. I have told him one rule at his PR, and yet he ignored/denied it.

Comment (zloyden) [reply EnglishFilter/sections/specific.txt:266]: Not reproduced on my end, but I'd add `||amazonaws.com/brand/$domain=hltv.org`.

Comment (wout-codes) [reply EnglishFilter/sections/specific.txt:266]: Yeah, I agree.

File: EnglishFilter/sections/specific.txt [hunk 2]
```diff
@@ -263,6 +263,8 @@ feetporno.com,familyporn.tv,freehardcore.com,webvoyeur.com,moviesand.com,momvids
 !
 !
 !
+||amazonaws.com/brand/$domain=hltv.org
+hltv.org##.featured-matches-top
```
[the remark this answers is not in the document]
Comment (wout-codes) [reply EnglishFilter/sections/specific.txt:267]: There is no such string named as `pari-match`.

<details>

image

</details>

[the remark this answers is not in the document]
Comment (zloyden) [reply EnglishFilter/sections/specific.txt:267]: Can you provide a link? Also we should add `hltv.org##.ggbe`

[the remark this answers is not in the document]
Comment (wout-codes) [reply EnglishFilter/sections/specific.txt:267]: Link for what?

[the remark this answers is not in the document]
Comment (zloyden) [reply EnglishFilter/sections/specific.txt:267]: Move your rules to `hltv.org##` rules.

[the remark this answers is not in the document]
Comment (wout-codes) [reply EnglishFilter/sections/specific.txt:267]: Do you want to know something?

[the remark this answers is not in the document]
Comment (wout-codes) [reply EnglishFilter/sections/specific.txt:267]: Useless:
```
hltv.org##div[class*="ASdvuasdr123Gazx"]
hltv.org##.thunderpick-firstcol-box
hltv.org##.pari-match
```

[the remark this answers is not in the document]
Comment (zloyden) [reply EnglishFilter/sections/specific.txt:267]: No, two of them are relevant, this `hltv.org##.thunderpick-firstcol-box` @stanislav-atr can reproduce.

[the remark this answers is not in the document]
Comment (wout-codes) [reply EnglishFilter/sections/specific.txt:267]: Okay, then.

[the remark this answers is not in the document]
Comment (zloyden) [reply EnglishFilter/sections/specific.txt:267]: The last thing you should do - sort them into a single list.

[the remark this answers is not in the document]
Comment (wout-codes) [reply EnglishFilter/sections/specific.txt:267]: I can't find the other elements for some reasons. Could you please check and merge?

[the remark this answers is not in the document]
Comment (zloyden) [reply EnglishFilter/sections/specific.txt]: Nvm, I did it. Thanks for your participation )

**Pull request merged**